### PR TITLE
chore: updated signup page

### PR DESCRIPTION
**Pull request by arjunbharti**
Added first name and last name input in signup form
Screenshot from 2022-02-20 20-40-17

Preview: https://deploy-preview-7--fakesea.netlify.app/pages/signup.html

### CODE REVIEW

**Review by sameep-s — APPROVED** [review]
Looks Good you can merge.

File: styles/login-and-signup.css [hunk 1]
```diff
@@ -59,4 +59,5 @@
     height: 34.375rem;
```
Comment (akash-1318) [inline styles/login-and-signup.css:59]: Is height needed here because it will take height according to the content?

**Review by akash-1318 — COMMENTED** [review]
Just one point added else everything looks good.

File: styles/login-and-signup.css (hunk 1 above)
Comment (arjunbharti) [reply styles/login-and-signup.css:59]: Yup, the card is getting sticked with the bottom. So, i have write height here.

**Pull request merged**